libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -302,6 +302,13 @@ ofFbo::ofFbo(const ofFbo & mom){
     }
 }
 
+//--------------------------------------------------------------
+void ofFbo::operator+=(const ofFbo & f){
+	this->begin();
+	f.draw(0,0);
+	this->end();
+}
+
```

Comment:
might want to add a check that if the fbo isn't allocated you can an error/warning 
or you could have if it is not allocated it gets allocated to match first?